parser_library/src/lsp/item_convertors.cpp

```diff
@@ -249,6 +250,31 @@ std::string get_macro_documentation(const text_data_view& text, size_t definitio
     return result;
 }
 
+std::string get_logical_line(const text_data_view& text, size_t definition_line)
+{
+    size_t end_line = definition_line;
+    for (size_t doc_limit = 20; doc_limit && end_line < text.get_number_of_lines(); --doc_limit)
+    {
+        if (!is_continued_line(text.get_line(end_line)))
+            break;
+        ++end_line;
+    }
+    ++end_line;
```

Comment:
I was a bit confused how the `for loop` on lines 269 - 272 could work as I implicitly thought that `end_line` meant last line of the statement, not the line after the statement, and didn't realize for a moment that `end_line` is incremented here.

Not saying, that it needs to be changed or anything - just noting this observation